### PR TITLE
Feature/add option to only store and retrieve csv files locally

### PR DESCRIPTION
**Pull request by ppmarkus**
I like the new feature to save as CSV but this only works with an Amazon S3 account.
Considering the file already gets saved to the server it'd be convenient if that file could be retrieved as a FileResponse directly without storing the intermediate result on the cloud. We for one are not interested in storing the created CSV files.

This feature adds a config `ONLY_STORE_CSV_FILES_LOCALLY` to set whether the user is fine only saving to S3.

Basically I would like it to work even if no S3_AWS credentials set up. This config is a quick means to check whether that can be bypassed. 

Then the result can be retrieved using the a new api call `get_response_local_file` which just creates a FileResponse from the local file.

### CODE REVIEW

**Comment by ppmarkus** [comment]
Appologies, I didn't rebase correctly. I will close this and create another one.
